src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml

```diff
@@ -14719,7 +14719,22 @@
                           <FontIcon Glyph="&#xE8EE;" />
                         </AppBarButton.Icon>
                       </AppBarButton>
-                      <AppBarButton x:Name="PlaybackRateButton" Style="{StaticResource AppBarButtonStyle}" MediaTransportControlsHelper.DropoutOrder="10" Visibility="Collapsed">
+                      <AppBarButton x:Name="PlaybackRateButton" Style="{StaticResource AppBarButtonStyle}" MediaTransportControlsHelper.DropoutOrder="10">
```

Comment:
Let's not change the style of the control (it's the official one), it's likely that the flyout is created from the code.

Reply:
Done.